### ISSUE TITLE
Change text on "choose a way to log in" page on dev

### ISSUE THREAD

**Issue by amandafrench** [VTUL/vtechworks#212]
The 'login' page currently at http://vtechworks.dev.lib.vt.edu/login will need to be modified. Current version looks like this:
![screenshot 2016-03-01 13 10 39](https://cloud.githubusercontent.com/assets/227453/13436864/4ee46fe6-dfaf-11e5-929c-f5b6514f1b22.png)

The text on the page should be as follows: 

> # Log in
> ### \* [Log in with your Virginia Tech PID ](url)
> 
> Other ways to log in 
> - [Log in with your existing VTechWorks account](url)
> - [Register a new VTechWorks account](for non-VT users) 
> 
> Registering for a new VTechWorks account will enable you to subscribe to collections so that you receive emails when new content is added; you will not be able to deposit material or to read material restricted to Virginia Tech. If you are not a member of the Virginia Tech community and would like to deposit material in VTechWorks, please write [vtechworks@vt.edu](url) with the reason for your request.  
> 
> Items you deposit are public by default; please make sure not to deposit material containing information you wish to keep confidential, such as your email address.

And then we should probably take the "register new user" stuff off the password-login page pictured below, although that's not essential. 

![screenshot 2016-03-01 13 56 32](https://cloud.githubusercontent.com/assets/227453/13438107/8e8e16a0-dfb5-11e5-9511-69213386a191.png)

**Comment by amandafrench** [comment]
Meant to mention that as best I can tell, this page isn't in source control, so we can't edit it on our local VMs.

**Comment by keithgee** [comment]
@amandafrench, you're right that there's no html file for this login choosing page!!

The code that generates this is in[ LoginChooser.java](https://github.com/VTUL/vtechworks/blob/9b5ddcfb65576adfdb13fd2bb75221270ec1b4d7/dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/eperson/LoginChooser.java).

Two options are:
1.  Create a maven overlay for a customized version of LoginChooser.java, and also customize related messages in the messages.xml.
2. Have the login page load a static file as part of its contents. This static file will have your customized login chooser page, which we will create and add to source control. I don't know if we can go with pure html, but this should work at least with a file in the same format as news-xmlui.xml. How about login-chooser.xml?

I haven't fully tested either option, but I'm leaning toward option 2. 

An anticipated advantage of approach 2 is that once this is finished, we'll actually have a page in source control that corresponds to the loginchooser page, and one that will be easier to modify in the future. Because look at the weirdness of that LoginChooser.java file from option number 1!

 An anticipated disadvantage of approach 2 is that if we ever internationalize VTechWorks, we'll have to handle this page differently than other pieces of DSpace, which use messages.xml for text values.

**Comment by mello99** [comment]
Okay, is it fair to say that we're not testing the the Shibboleth login on dev right now, @amandafrench and @keithgee ? Until we get the homepage issue resolved, that is. I ask because 1) I read your discussion, and 2) I tried logging-in to dev and no longer saw the Shibboleth notice (see below). Thanks, take care.

<img width="1235" alt="screen shot 2016-03-03 at 5 29 15 pm" src="https://cloud.githubusercontent.com/assets/5408616/13511944/9c19f012-e165-11e5-9b6e-8f98e82f8673.png">

**Comment by amandafrench** [comment]
This is fixed, @mello99 -- looks like the new deployment branch I set up yesterday overwrote the Shibboleth stuff that @kayiwa and @keithgee were doing. But they've re-implemented Shib on the dev server; give it another shot.

**Comment by amandafrench** [comment]
Shibboleth implementation is on hold till further notice.
